packages/ui/src/coreModules/search/utilities/queryBuilderFactory/utilities/factories/featureRangeSpecification.js

```diff
@@ -1,27 +1,24 @@
-import createTagTypesSpecification from './tagTypesSpecification'
-
 export default function featureRangeSpecification({
+  fieldName,
   rangeFilterFunctionName,
+  rangeTypeAggregateTagsFunctionName,
   sectionName,
-  fieldName,
-
-  rangeTypesAggregateTagsFunctionName,
-  rangeTypesMatchFilterFunctionName,
 }) {
-  const valueFieldName = `${fieldName}.rangeValue`
-  const rangeTypesFieldName = `${fieldName}.rangeTypes`
-
-  const specifications = []
+  const stringAggregation = () => {
+    return {
+      aggregationFunction: rangeTypeAggregateTagsFunctionName,
+    }
+  }
 
   const rangeFilter = (options = {}) => {
-    const {
-      fieldValue: { min, max } = {},
-      sectionValues: { rangeTypes, rangeUnit } = {},
-    } = options
+    const { sectionValues: { rangeType, rangeValue, rangeUnit } = {} } = options
```

Comment:
Here you should use fieldValue instead of sectionValue

Reply:
https://github.com/DINA-Web/dina-collections/pull/392/commits/2d29812a794ba5b4a663a960f9e7b1b411f7baa9